docs_source_files/content/webdriver/browser_manipulation.de.md

```diff
@@ -1229,7 +1237,21 @@ ele.screenshot('./image.png')
 
 driver.quit()
   {{< / code-panel >}}
-  {{< code-panel language="csharp" >}} // code sample not available please raise a PR {{< / code-panel >}}
+  {{< code-panel language="csharp" >}}
+    using OpenQA.Selenium;
+    using OpenQA.Selenium.Chrome;
+    using OpenQA.Selenium.Support.UI;
+
+    // Webdriver
+    var driver = new ChromeDriver();
+    
+    // Fetch element using FindElement
+    var webElement = driver.FindElement(By.XPath(@"(//*[@id=""messages""])/div[last() - 1]"));
```

Comment:
Couple of things,

1. like all other code samples, can we need to navigate to URL (http://www.example.com/) and 
2. like other code samples can we change to driver.FindElement(By.CssSelector("h1"));